src/utils/events/event-manager.js

```diff
@@ -136,6 +142,8 @@ export default class EventManager {
    * Process the event deregistration for a single event + handler.
    */
   _removeEventHandler(event, handler) {
+    let success = false;
```

Comment:
nit: call this `eventHandlerRemoved` to be a bit more explicit about what succeeded